### DISCUSSION TITLE
how to set git diff and merge to use zed?

### DISCUSSION THREAD

**Discussion by theArianit** [zed-industries/zed#15874] [Feature Requests]
How to set git diff, difftool, merge and mergetool to use Zed?

**Comment by rambip** [comment]
From what I understand, zed can't be used as a mergetool right yet.

But this comment seems to indicate that things could change very soon: https://github.com/zed-industries/zed/issues/34813#issuecomment-3603689143

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
